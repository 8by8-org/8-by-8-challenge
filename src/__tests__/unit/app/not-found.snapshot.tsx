import { render, cleanup } from '@testing-library/react';
import NotFound from '@/app/not-found';

describe('Not Found', () => {
    afterEach(cleanup);

    it('renders 404 page', () => {
        const { container } = render(<NotFound />);
        expect(container).toMatchSnapshot();
    });
});
