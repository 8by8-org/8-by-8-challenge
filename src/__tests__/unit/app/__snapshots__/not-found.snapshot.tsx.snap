// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found renders 404 page 1`] = `
<div>
  <div
    class="outer_container"
  >
    <div
      class="inner_container"
    >
      <section
        class="container"
      >
        <img
          alt="404, page not found"
          data-nimg="1"
          decoding="async"
          height="40"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
          style="color: transparent;"
          width="40"
        />
        <h1>
          Not Found
        </h1>
      </section>
    </div>
  </div>
</div>
`;
